include/openenclave/enclave.h

```diff
@@ -173,6 +173,21 @@ void* oe_host_realloc(void* ptr, size_t size);
  */
 void* oe_host_calloc(size_t nmemb, size_t size);
 
+/**
+ * Fill block of host memory.
+ *
+ * This function sets the first **num** bytes of the block of host memory
+ * pointed by **ptr** to value specified by **value**.
+ *
+ * @param ptr The host memory block to fill.
```

Comment:
we should be using [in,out] style doxygen annotations as well.  It's easiest to just start doing it for all new changes than to have to retroactively figure them out later when you actually know the answer for these lines now.